src/single-spa-vue.js

```diff
@@ -145,6 +151,11 @@ function unmount(opts, mountedInstances, props) {
     instance.vueInstance.$el.innerHTML = "";
     delete instance.vueInstance;
 
+    // detach Vue instance to the vue devtools
```

Comment:
This could be a problem when there is a shared instance of Vue between several microfrontends ([example](https://github.com/vue-microfrontends/shared-dependencies/)). In that case, we wouldn't want to detach from the devtools when only one application unmounts - we'd want to only do so when all of them unmount, or perhaps just never.

I think it would be best to remove the unmounting logic for devtools entirely - doesn't hurt to keep the Vue instance attached to the devtools, I think?